flixel/addons/editors/tiled/TiledTileSet.hx

```diff
@@ -25,9 +25,11 @@ class TiledTileSet
 	public var numCols:Int;
 	
 	public var properties:TiledPropertySet;
-	
 	public var tileProps:Array<TiledPropertySet>;
 	
+	public var imagesSources:TiledImageSet;
```

Comment:
Should be `imageSources` and `tileImageSources`, the duplicate plural looks strange.